attributes/default.rb

```diff
@@ -518,6 +518,8 @@
 default['cluster']['fsx_fs_ids'] = ''
 default['cluster']['fsx_dns_names'] = ''
 default['cluster']['fsx_mount_names'] = ''
+default['cluster']['fsx_fs_types'] = ''
```

Comment:
Just for my better understanding.
If the cluster config file has the following shared file systems: `FsxLustre,FsxOntap,FsxOpenZfs,FsxLustre,FsxOntap,FsxOpenZfs,FsxLustre`, will the two new attributes have the following values?

`fsx_fs_types = 'LUSTRE,ONTAP,OPENZSF,LUSTRE,ONTAP,OPENZSF,LUSTRE'`
`fsx_volume_junction_paths = ',/a/junction/path_1,/a/junction/path_2,,/a/junction/path_3,/a/junction/path_4,'`